gem/gem.py

```diff
@@ -782,6 +783,79 @@ def __new__(cls, i, j):
         return self
 
 
+# TODO: inverse should be different from comp tensor
+# in the sense that it does not take a local to a global view of a matrix
+# but instead eats a global view and spits out another global view
+# it does not transformation between free indices and shape so you would think it does not need a multiindex
+# but the multiindex is necessary to feed in the global view of the tensor
+# it can still have free_indices inherited by the  aggregate (?)
```

Comment:
Can you explain what is meant by _local_ and _global_ view of a matrix?  I have no idea what this text means...

Reply:
Okay, I think I'm staring to get it... by *global* you probably mean shape and by *local* you mean free indices.

Reply:
So, yes, I agree that `ComponentTensor` is different from `Inverse`.

Reply:
Yes, sorry for terrible phrasing.